bootstrap/cmd/bootstrap/app/ksServer.go

```diff
@@ -1145,6 +1205,14 @@ func finishDeployment(svc KsService, req CreateRequest, dmDeploy *deploymentmana
 		return
 	}
 
+	log.Infof("Installing Istio...")
```

Comment:
Do we have a flag to control whether or not to install istio?

Reply:
Istio will be required for multi-user support, so I think we always want to install istio?
@jlewi WDYT?

Reply:
Ideally we'd like to specify this declaratively somewhere but I think that involves a bigger change. 

As a short term solution, can we condition this based on the Kubeflow version? For 0.4 we shouldn't install ISTIO by default because that would be a breaking change. But for newer versions of Kubeflow we should.

Longer term, do we check in the ISTIO manifests into our repo? Maybe that's something we should consider doing. This way each version of Kubeflow would have an associated set of versioned manifests just like we do for deployment manager configs.

Reply:
I made it a flag and default to false now.
We do check in istio manifest